versions.tf

```diff
@@ -2,8 +2,8 @@ terraform {
   required_version = ">= 0.12.0, < 0.14.0"
```

Comment:
can you remove the upper version pinning here too?

Reply:
@woz5999 upper version limit removed.